stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -134,6 +136,14 @@ internal class ChannelControllerImpl(
         .map { it.values.sortedBy { user -> user.createdAt } }
         .asLiveData()
 
+    val _typingUsers: StateFlow<List<User>> = _typing.map {
+        it.values
+            .sortedBy(ChatEvent::createdAt)
+            .mapNotNull { event ->
+                (event as? TypingStartEvent)?.user ?: (event as? TypingStopEvent)?.user
```

Comment:
If the event is `TypingStopEvent` it indicate that the user has stopped to write, Shouldn't it be removed?

Reply:
Thierry wrote this and I forgot about it. I'll check this out.

Reply:
I think Thierry changed this. I'll check it out.